backend.prod.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM node:alpine3.17 AS builder
+WORKDIR /app
+
+COPY packages/server/package.json ./
+COPY yarn.lock ./
+RUN yarn remove node-cron @joystream/types
+RUN yarn --immutable
+
+COPY tsconfig.json ./base.tsconfig.json
+COPY packages/server/tsconfig.json ./
+RUN sed -i 's/"extends":.*/"extends": ".\/base.tsconfig.json",/' tsconfig.json
+
+COPY packages/server/prisma ./
+COPY packages/server/codegen.ts ./
+COPY packages/server/src ./src
+RUN yarn build:local
+
+RUN rm -rf node_modules
+ENV NODE_ENV=production
+RUN yarn --prod --immutable
+RUN yarn prisma generate
+
+FROM alpine:3.17
+WORKDIR /app
+RUN set -eux; \
+    adduser -u $(getent group www-data | cut -d: -f3) -S -D -G www-data www-data; \
+    apk add --no-cache nodejs
```

Comment:
Why build own nodejs image when there are prebuilt available?

Reply:
I think to reduce the size again (the `node:alpine` images are bigger than manually installing nodejs) but TBH I don't remember how significant that was I might have gone overboard here !

Reply:
The gain was `118MB` but I think it's still worth using prebuilt image to make sure the node version is the same as the one it builds from.